docs/reference/python/index.md

```diff
@@ -0,0 +1,53 @@
+# Getting Started
+
+We use Python 3 to write the majority of our software at UBC Sailbot. Therefore, it is critical that you get familiar
+with the language and be able to use it to get your assigned tasks completed.
+
+## Installation
```

Comment:
I don't think we need this section. I can only imagine us recommending using sailbot_workspace, WSL Ubuntu, or native Ubuntu, which all comes with Python. For anything else it's the person's responsibility to figure out.